Solutions/DNS Domain Solution/SolutionMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "publisherId": "azuresentinel",
+    "offerId": "azure-sentinel-solution-DNS-preview",
+    "firstPublishDate": "2023-01-14",
+    "lastPublishDate": "2023-01-14",
+    "providers": [ "Microsoft" ],
+    "categories": {
+      "domains": [ "Security – DNS" ]
```

Comment:
@vakohl, can you please use the domain mentioned in this doc (https://learn.microsoft.com/en-us/azure/sentinel/sentinel-solutions), thanks.

Reply:
done